common4j/src/main/com/microsoft/identity/common/java/logging/DiagnosticContext.java

```diff
@@ -70,6 +70,11 @@ public IRequestContext getRequestContext() {
         return REQUEST_CONTEXT_THREAD_LOCAL.get();
     }
 
+    public String getThreadCorrelationId() {
+        return getRequestContext().containsKey(DiagnosticContext.CORRELATION_ID)
```

Comment:
Don't really need to do look-up twice (contains and get are two look-ups). You can just do `get` directly and look at the return value (null is returned if the value was not present).